pallets/staking/src/lib.rs

```diff
@@ -0,0 +1,1256 @@
+// This file is part of Substrate.
```

Comment:
We should move staking back to polkadot-sdk

Reply:
This will be done at the end to simplify dev